custom-routes-vis/src/main/webapp/script.js

```diff
@@ -23,10 +23,13 @@ const MarkerNames = {
   DESTINATION: 'destination',
 };
 
+// Array holding origin and destination markers.
 let originDestinationMarkers = [];
+// Array holding routes displayed on the map.
 let displayedRoutes = [];
-const routeColors = ['#ff0000', '#eb8f1e', '#3299d1'];
-let selectedRouteNum = 0;
+// Colors of the routes displayed.
+const ROUTE_COLORS = ['#ff0000', '#eb8f1e', '#3299d1'];
```

Comment:
can you give names to the colors instead?

Reply:
Can I just add a comment on their names? Because I need them to be in a list to get them according to route number (see line 255)

Reply:
Sounds good.